### DISCUSSION TITLE
News Cache [2023-02-27]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#1644] [Announcements]
### 🚢  New Feature

- [Run scheduled jobs with Vercel Cron Jobs and Vercel Functions](https://vercel.com/changelog/run-scheduled-jobs-with-vercel-cron-jobs-and-vercel-functions)

### 💡 Inspiration

- [From monolith to composable: equipping a financial services IPO](https://vercel.com/customers/from-monolith-to-composable-equipping-a-financial-services-ipo)
- [How Makeswift improved CI speed by 65% with Turborepo](https://vercel.com/customers/how-makeswift-improved-ci-speed-by-65-with-turborepo)
- [How a global agency built a web innovation engine in two months](https://vercel.com/customers/how-a-global-agency-built-a-web-innovation-engine-in-two-months)

### 📚 Resources

- [Your guide to headless commerce](https://vercel.com/blog/your-guide-to-headless-commerce)
- [Introducing Vercel Cron Jobs](https://vercel.com/blog/cron-jobs)
- [Vercel Cache API: A progressive cache, integrated with Next.js](https://vercel.com/blog/vercel-cache-api-nextjs-cache)

**Comment by marcelmusiol** [comment]
Will the Vercel Cache API eventually be available for other frameworks too (Remix for example)?

**Reply by amyegan** [reply]
Excellent question, Marcel! Vercel Cache API is in beta and currently only works with Next. But wider availability is planned as development progresses
